src/Entity.php

```diff
@@ -3246,146 +3268,174 @@ public static function getUsedConfig($fieldref, $entities_id, $fieldval = '', $d
         }
 
-        if (strstr($url, "[TICKET_NAME]")) {
-            $url = str_replace("[TICKET_NAME]", urlencode($ticket->fields['name']), $url);
+        if (strstr($url, "[{$tag_prefix}_ID]")) {
+            $url = str_replace("[{$tag_prefix}_ID]", $item->fields['id'], $url);
         }
 
-        if (strstr($url, "[TICKET_CREATEDATE]")) {
-            $url = str_replace("[TICKET_CREATEDATE]", $ticket->fields['date'], $url);
+        if (strstr($url, "[{$tag_prefix}_NAME]")) {
+            $url = str_replace("[{$tag_prefix}_NAME]", urlencode($item->fields['name']), $url);
         }
 
-        if (strstr($url, "[TICKET_SOLVEDATE]")) {
-            $url = str_replace("[TICKET_SOLVEDATE]", $ticket->fields['solvedate'], $url);
+        if (strstr($url, "[{$tag_prefix}_CREATEDATE]")) {
+            $url = str_replace("[{$tag_prefix}_CREATEDATE]", $item->fields['date'], $url);
         }
 
-        if (strstr($url, "[REQUESTTYPE_ID]")) {
-            $url = str_replace("[REQUESTTYPE_ID]", $ticket->fields['requesttypes_id'], $url);
+        if (strstr($url, "[{$tag_prefix}_SOLVEDATE]")) {
+            $url = str_replace("[{$tag_prefix}_SOLVEDATE]", $item->fields['solvedate'], $url);
         }
 
-        if (strstr($url, "[REQUESTTYPE_NAME]")) {
-            $url = str_replace(
-                "[REQUESTTYPE_NAME]",
-                urlencode(Dropdown::getDropdownName(
-                    'glpi_requesttypes',
-                    $ticket->fields['requesttypes_id']
-                )),
-                $url
-            );
+        if ($item::getType() === 'Ticket') {
+            if (strstr($url, "[REQUESTTYPE_ID]")) {
+                $url = str_replace("[REQUESTTYPE_ID]", $item->fields['requesttypes_id'], $url);
+            }
+
+            if (strstr($url, "[REQUESTTYPE_NAME]")) {
+                $url = str_replace(
+                    "[REQUESTTYPE_NAME]",
+                    urlencode(Dropdown::getDropdownName(
+                        'glpi_requesttypes',
+                        $item->fields['requesttypes_id']
+                    )),
+                    $url
+                );
+            }
         }
 
-        if (strstr($url, "[TICKET_PRIORITY]")) {
-            $url = str_replace("[TICKET_PRIORITY]", $ticket->fields['priority'], $url);
+        if (strstr($url, "[{$tag_prefix}_PRIORITY]")) {
+            $url = str_replace("[{$tag_prefix}_PRIORITY]", $item->fields['priority'], $url);
         }
 
-        if (strstr($url, "[TICKET_PRIORITYNAME]")) {
+        if (strstr($url, "[{$tag_prefix}_PRIORITYNAME]")) {
             $url = str_replace(
                 "[TICKET_PRIORITYNAME]",
-                urlencode(CommonITILObject::getPriorityName($ticket->fields['priority'])),
+                urlencode(CommonITILObject::getPriorityName($item->fields['priority'])),
                 $url
             );
         }
 
         if (strstr($url, "[TICKETCATEGORY_ID]")) {
```

Comment:
`TICKETCATEGORY_*` is not documented anymore in form file. Maybe a migration should replace it by `ITILCATEGORY_*` in `glpi_entities` table, and a deprecation message should be added here (and added in changelog).